4001-5000/LIT4867Theodore.xml

```diff
@@ -63,18 +66,19 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
             <listRelation>
                <relation name="ecrm:P129_is_about" active="LIT4867Theodore" passive="PRS9475Theodoru"/>
                <relation name="saws:isAttributedToAuthor" active="LIT4867Theodore" passive="PRS5720JohnChr"/>
+               <relation name="ecrm:CLP46i_may_form_part_of" active="LIT6463Letter" passive="LIT1151Asceti"/>
             </listRelation>
```

Comment:
Please, the relation "formerlyListeAs" as in the GL

Reply:
Done